Swat/SwatXHTMLTextarea.php

```diff
@@ -58,29 +47,27 @@ public function process()
 
             $xhtml_template =
                 $doctype .
-                <<<XHTML
-                <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
-                	<head>
-                		<meta http-equiv="content-type" content="text/html; charset=UTF-8" />
-                		<title>SwatXHTMLTextarea Content</title>
-                	</head>
-                	<body>
-                		<div>
-                		%s
-                		</div>
-                	</body>
-                </html>
-
-                XHTML;
+                <<<'EOB'
```

Comment:
Why EOB? XHTML seems more correct.

Reply:
Code editors will use the delimiters to do language injection. I think it still makes sense to use XHTML so you get auto-syntax checks of the sub-language in various editors.

Reply:
Reverting this change, and will update the php-cs-fixer ruleset to ignore the choice of delimiter string.